docs/user-guide/ui.md

```diff
@@ -1,120 +1,120 @@
+
+- **Environment variables**: Kubernetes exposes Services through [environment variables](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/design/expansion.md). You can compose environment variable or pass arguments to your commands using the values of environnment variables. They can be used in applications to find a Service. Environment variables are also useful for decreasing coupling and the use of workarounds. Values can reference other variables using the `$(VAR_NAME)` syntax.
+
+#### Uploading a YAML or JSON file
+
+Kubernetes supports declarative configuration. In this style, all configuration is stored in YAML or JSON configuration files using the Kubernetes' [API](http://kubernetes.io/v1.1/docs/api.html) resource schemas as the configuration schemas.
+
+As an alternative to specifying application details in the deploy wizard, you can define your Replication Controllers and Services in YAML or JSON files, and upload the files to your Pods:
+
+![Kubernetes Dashboard deploy from file upload](/images/docs/ui-dashboard-deploy-file.png)
+
+### Applications view
+
+As soon as applications are running on your cluster, the initial view of the Dashboard defaults to showing an overview of them, for example:
+
+![Kubernetes Dashboard applications view](/images/docs/ui-dashboard-rcs.png)
+
+Individual applications are shown as cards - where an application is defined as a Replication Controller and its corresponding Services. Each card shows the current number of running and desired replicas, along with errors reported by Kubernetes, if any.
+
+You can view application details (**View details**), make quick changes to the number of replicas (**Edit pod count**) or delete the application directly (**Delete**) from the menu in each card's corner:
 
 ![Kubernetes Dashboard deploy form file upload](/images/docs/ui-dashboard-cards-menu.png)
 
 #### View details
 
-Selecting this option from the card menu will take you to the following page where you can view more information about the Pods that make up your application:
+Selecting this option from the card menu will take you to the following page where you can view more information about the Pods that make up your application:
+
+![Kubernetes Dashboard application detail](/images/docs/ui-dashboard-rcs-detail.png)
+
+The **EVENTS** tab can be useful for debugging flapping applications.
+
+Clicking the plus sign in the right corner of the screen leads you back to the page for deploying a new application.
+
+#### Edit pod count
+
+If you choose to change the number of Pods, the respective Replication Controller will be updated to reflect the newly specified number.
+
+#### Delete
 
-![Kubernetes Dashboard application detail](/images/docs/ui-dashboard-rcs-detail.png)
+Deleting a Replication Controller also deletes the Pods managed by it. It is currently not supported to leave the Pods running.

```

Comment:
You can remove: "It is currently not supported to leave the Pods running." 

This is implied, plus that's in passive voice. :)